htdocs/don/class/paymentdonation.class.php

```diff
@@ -130,7 +130,7 @@ public function create($user, $notrigger = false)
 		$now = dol_now();
 
 		// Validate parameters
-		if (!$this->datepaid) {
+		if (!$this->datep) {
```

Comment:
Line 192, we use this->datepaid.
So if we change here, we must also change on line 192